src/components/panes/ActionReminderPane.scss

```diff
@@ -37,23 +37,25 @@
 
             .Avatar {
                 width: 5em;
-                height: auto;
+                height: 5em;
+                float: left;
             }
 
-            .Person {
-                position: absolute;
-                top: 10px;
-                left: 60px;
+            .ActionReminderPane-remindedInfo {
+                float: left;
+                margin-left: 1em;
+            }
 
+            .Person {
                 cursor: pointer;
-                font-size: 1.1em;
+                font-size: 1.4em;
                 font-weight: bold;
```

Comment:
In most cases like this, we use a larger font-size rather than bold. I suggest removing the `font-weight` and do `font-size: 1.7em`.